Resources/config/services.php

```diff
@@ -157,4 +158,17 @@
             new Reference('gesdinet.jwtrefreshtoken.refresh_token_manager'),
         ])
         ->tag('console.command');
+
+    $services->set(LogoutEventListener::class)
+        ->args([
+            new Reference('gesdinet.jwtrefreshtoken.refresh_token_manager'),
+            new Reference('gesdinet.jwtrefreshtoken.request.extractor.chain'),
+            new Parameter('gesdinet_jwt_refresh_token.token_parameter_name'),
+            new Parameter('gesdinet_jwt_refresh_token.cookie'),
+            new Parameter('gesdinet_jwt_refresh_token.logout_firewall_context'),
+        ])
+        ->tag('kernel.event_listener', [
```

Comment:
Would've preferred to set `'dispatcher' => 'security.event_dispatcher.{$logout_firewall}'` to avoid calling the `LogoutEventListener` altogether instead of checking the firewall context within the listener, however I couldn't figure out how to reference a parameter value from this scope.

Reply:
The other option I can think of is to register the service in the container extension class, that way you have the bundle config available.

But, I don't think it's worth the effort at the moment.  The bundle already has a bunch of other quirks that make it difficult to use in a multi-firewall setup (Sylius is probably the most practical example I can think of where it has multiple firewalls, you wouldn't be able to handle refresh tokens for the shop and admin API contexts separately).  So it's fine for now to go with this and later on if things are reworked then the event listener could be attached to the right dispatcher(s).